buildtools/cli.go

```
@@ -1212,6 +1213,42 @@ func pythonCmd(c *cli.Context, projectType project.ProjectType) error {
 		return cliutils.WrongNumberOfArgumentsHandler(c)
 	}
 
+	// FlexPack native mode for Poetry (bypasses config file requirements)
+	if os.Getenv("JFROG_RUN_NATIVE") == "true" && projectType == project.Poetry {
+		log.Debug("Routing to Poetry native implementation")
+		args := cliutils.ExtractCommand(c)
+		filteredArgs, buildConfiguration, err := build.ExtractBuildDetailsFromArgs(args)
+		if err != nil {
+			return err
+		}
+		cmdName, poetryArgs := getCommandName(filteredArgs)
+
+		// Execute native poetry command directly (similar to Maven FlexPack)
+		log.Info(fmt.Sprintf("Running Poetry %s.", cmdName))
+		poetryCmd := exec.Command("poetry", append([]string{cmdName}, poetryArgs...)...)
+		poetryCmd.Stdout = os.Stdout
+		poetryCmd.Stderr = os.Stderr
+		if err := poetryCmd.Run(); err != nil {
+			return fmt.Errorf("poetry %s failed: %w", cmdName, err)
+		}
+
+		// Collect build info if build parameters provided
+		if buildConfiguration != nil {
+			buildName, err := buildConfiguration.GetBuildName()
+			if err == nil && buildName != "" {
+				workingDir, _ := os.Getwd()
+				if err := buildinfo.GetPoetryBuildInfo(workingDir, buildConfiguration); err != nil {
+					log.Warn("Failed to collect Poetry build info: " + err.Error())
+				} else {
+					buildNumber, _ := buildConfiguration.GetBuildNumber()
```

are we ignoring error here?